sysbuild/Kconfig.suit

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "SUIT"
+
+menuconfig SUIT_ENVELOPE
+	bool "Create SUIT envelope"
+	help
+	  Enable DFU SUIT envelope creation
+	default y if $(BOARD) = "nrf54h20dk_nrf54h20_cpuapp"
+
+if SUIT_ENVELOPE
+
+config SUIT_ENVELOPE_SIGN
+	bool "Sign created SUIT envelope"
+	help
+	  Sign created SUIT envelope by external script
+	default n
+
+config SUIT_ENVELOPE_SEQUENCE_NUM
+	int "Sequence number of the generated SUIT manifest"
+	range 0 2147483647
+	default 1
+
+config SUIT_ENVELOPE_ROOT_TEMPLATE
+	string "Path to the default root envelope template"
+	default "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/ncs/root_with_binary_nordic_top.yaml.jinja2"
```

Comment:
This does not belong in Kconfig, probably better to either have a specific file name in the application folder which will be used if it exists or the default if not, or have the user provide a filename with a variable e.g. `-DSUIT_TEMPLATE=<file>` but not using Kconfig

Reply:
I do not see the `prj.conf` as a problem - you still can use those variables in Kconfigs as well as testcase/sample.yaml files. The `prj.conf` is used only for the default build.
I agree though that using Kconfig is not necessary here as long as we can modify the root template in yaml files and through command line options. This value is used only by the CMake scripts, no C code depends on it.

Reply:
Could this change be discussed later? This PR does not change any behavior at the high level and the Kconfigs here are only moved from normal kconfig into sysbuild kconfig.

Reply:
Please create a JIRA